consensus/istanbul/backend/backend.go

```diff
@@ -425,3 +425,33 @@ func (sb *backend) HasBadProposal(hash common.Hash) bool {
 	}
 	return sb.hasBadBlock(hash)
 }
+
+// CreateGovernanceState is a function that creates governanceState manually
+// with given list of header 'headers'.
+// The number of last header(headers[len(headers)-1]) should be same with parameter 'num'.
+func (sb *backend) CreateGovernanceState(num uint64, headers []*types.Header) {
+	sb.governance.GovernanceVotes = governance.NewGovernanceVotes()
+	sb.governance.GovernanceTallies = governance.NewGovernanceTallies()
+
+	Votes := make([]governance.GovernanceVote, 0)
+	Tally := make([]governance.GovernanceTallyItem, 0)
+	for _, header := range headers {
```

Comment:
If there is a header that has `blockNumber % epoch == 0`, `votes` and `tallies` should be reset and votes results should be reflected because an epoch is ended.